connectors/activemq/src/main/java/forklift/connectors/ActiveMQConnector.java

```diff
@@ -100,6 +109,31 @@ public ForkliftConsumerI getTopic(String name)
     }
 
     @Override
+    public ForkliftConsumerI getConsumerForSource(SourceI source) throws ConnectorException {
```

Comment:
I think some javadoc on the interface would help here.  Explain how implementations should throw an exception if the requested source type is not supported by the connector.